### DISCUSSION TITLE
Send specific app to specific Desktop automatically

### DISCUSSION THREAD

**Discussion by milesromeno** [asmvik/yabai#2071] [Q&A] [answered]
I have some keybinds to spawn specific apps like my web browser and terminal. I like to keep particular apps on particular Desktop spaces for my workflow and muscle memory. Is there a way using yabai or skhd to make an application instantly move to a predefined desktop on spawning, or spawn to the desktop directly? Anyone doing anything similar?

**Comment by amlanjlahkar** [comment, marked as answer]
You can use application [rules](https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc#rule) for this.
For example, to always open safari on space 2
`yabai -m rule --add app="^Safari$" space=^2`
